website/manual.md

```diff
@@ -23,68 +51,59 @@ program, it is runnable with nothing more than
 Deno explicitly takes on the role of both runtime and package manager. It uses a
 standard browser-compatible protocol for loading modules: URLs.
 
-Deno provides security guarantees about how programs can access your system with
-the default being the most restrictive secure sandbox.
-
-Deno provides <a href="https://github.com/denoland/deno/tree/master/std">a set
-of reviewed (audited) standard modules</a> that are guaranteed to work with
-Deno.
+Among other things, Deno is a great replacement for utility scripts that may
+have been historically written with bash or python.
 
 ### Goals
 
-- Support TypeScript out of the box.
-
+1. Only ship a single executable (`deno`).
+2. Provide Secure Defaults
+   - Unless specifically allowed, scripts can't access files, the environment,
+     or the network.
+3. Browser compatible: The subset of Deno programs which are written completely
+   in JavaScript and do not use the global `Deno` namespace (or feature test for
+   it), ought to also be able to be run in a modern web browser without change.
+4. Be able to serve HTTP efficiently
+5. Provide a great developer experience include built-in tooling.
+6. Does not leak browser or V8 concepts into user land.
+
+### Comparison to Node.js
+
+- Deno does not use `npm`
+  - It uses modules referenced as URLs or file paths
+- Deno does not use a `package.json`
+  - This is a non-goal. There are effective patterns [citation needed] for
+    managing dependencies.
+- Deno provides different APIs than node.
+- Deno requires explicit permissions for file, network, and environment access.
+  - Node is less secure out of the box.
+- Deno always dies on uncaught errors.
 - Uses "ES Modules" and does not support `require()`. Like the browser, allows
   imports from URLs:
 
-  ```ts
-  import * as log from "https://deno.land/std/log/mod.ts";
-  ```
+```ts
+import * as log from "https://deno.land/std/log/mod.ts";
+```
+
+### Other key behaviors
 
 - Remote code is fetched and cached on first execution, and never updated until
   the code is run with the `--reload` flag. (So, this will still work on an
   airplane. See `~/.deno/src` for details on the cache.)
```

Comment:
```suggestion
   airplane.)
```
(This was already here)

Reply:
I'm not sure what you mean here. I didn't modify this.

Reply:
Yes, this was already here but it's outdated and may as well be addressed now.